web/src/main/webapp/dataViews/heavyHittersView.ts

```diff
@@ -78,36 +112,48 @@ export class HeavyHittersView extends RemoteTableObjectView {
         }
     }
 
+    public showSelected(): void {
+        let newPage2 = new FullPage("Frequent elements", "HeavyHitters", this.page);
```

Comment:
Watch out: the first argument of "FullPage" is used to compute a label in the help web page. Please make sure that the argument is correct.